oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/dsl/Sample.java

```diff
@@ -38,7 +38,7 @@
 @Getter
 public class Sample {
     final String name;
-    final ImmutableMap<String, String> labels;
+    final Map<String, String> labels;
```

Comment:
Why Sample's labels changeable now?

Reply:
> Why Sample's labels changeable now?

Not changeable, just replace `ImmutableMap` with `Map` interface that can accept other type of `Map`, similar to `Map<xxx, xxx> a = new HashMap<>()` instead of `HashMap<xxx, xxx> a = new HashMap<>()`

Reply:
If not really changeable, when have to change this? I think we should build a new `ImmutableMap` from a `Map`/`HashMap` in building Sample

Reply:
> I think we should build a new `ImmutableMap` from a `Map`/`HashMap` in building Sample

Correct me if I'm wrong, the only way to build a new `ImmutableMap` from a `Map` is `com.google.common.collect.ImmutableMap#copyOf(java.util.Map<? extends K,? extends V>)`, which needs to copy the entire map entries, does it work for you?

Reply:
Also, use `ImmutableMap` as the type doesn't allow use to use `Collections.unmodifiableMap()` to wrap the map

Reply:
Copy makes sense to me. I just don't want any further PR(not yours) adding same dynamic label mechanism but missed by the PMC.

Reply:
Done